corehq/apps/translations/app_translations/download.py

```diff
@@ -263,6 +264,13 @@ def _get_module_detail_no_items_text(langs, module):
     ]
 
 
+def _get_module_detail_select_text(langs, module):
+    short_detail = module.case_details.short
+    return [
```

Comment:
should this also check app version? Otherwise, it'll populate the exported file and won't be used if the app version doesn't support custom select text.

Reply:
That makes sense. It's not causing any harm, but it's better to do something similar to what you did with [no_item_text](https://github.com/dimagi/commcare-hq/blob/42efec5ff1bcaee0ab6e3a9b2e327f21675dafda/corehq/apps/translations/app_translations/download.py#L257).

Reply:
added here https://github.com/dimagi/commcare-hq/pull/33976/commits/fd0f80d5450e147331137bc8a96f01fb2a9cabae